modules/nw-sriov-runtime-config.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-net-attach.adoc
+
+[id="nw-sriov-runtime-config_{context}"]
+= Defining a runtime configuration for a network attachment
+
+When attaching a Pod to an additional network, you can specify a runtime configuration to make specific customizations for the Pod. For example, you can request a specific MAC hardware address.
+
+You specify the runtime configuration by setting an annotation in the Pod specification. The annotation key is `k8s.v1.cni.cncf.io/networks` and it accepts a JSON object describing the runtime configuration.
+
+[id="runtime-config-ethernet_{context}"]
+== Runtime configuration for an Ethernet-based SR-IOV attachment
+
+The following JSON describes the runtime configuration options for an Ethernet-based SR-IOV network attachment.
+
+[source,json]
+----
+[
+  {
+    "name": "<name>", <1>
+    "mac": "<mac_address>", <2>
+    "ips": ["<cidr_range>"] <3>
+  }
+]
+----
+<1> The name of the SR-IOV network attachment definition CR.
+<2> Optional: The MAC address for the SR-IOV device that is allocated from the resource type defined in the SR-IOV network attachment definition CR. To use this feature, you also must specify `{ "mac": true }` in the `SriovNetwork` object.
```

Comment:
I don't think that it's clear enough when you define the network objects that are mentioned in any of these optional parameters what these parameters actually let you do.